Cargo.toml

```diff
@@ -207,6 +207,8 @@ procinfo = { git = "https://github.com/tikv/procinfo-rs", rev = "6599eb9dca74229
 # After the PR to rust-rocksdb is merged, remember to comment this out and run `cargo update -p rocksdb`.
 # [patch.'https://github.com/tikv/rust-rocksdb']
 # rocksdb = { git = "https://github.com/your_github_id/rust-rocksdb", branch = "your_branch" }
+[patch.'https://github.com/tikv/rust-rocksdb']
+rocksdb = { git = "https://github.com/tabokie/rust-rocksdb", branch = "230720-fix-checkpoint-again" }
```

Comment:
Please use tikv/rust-rocksdb, could you send PR to tikv/rust-rocsdb first?

Reply:
https://github.com/tikv/rocksdb/pull/342